### PR TITLE
Totally steals trams from Paradise ♥

### PR DESCRIPTION
**Pull request by Nienhaus**
https://github.com/ParadiseSS13/Paradise/pull/1106

### CODE REVIEW

**Comment by TGameCo** [comment]
Have you tested these things yet?

**Comment by scrdest** [comment]
Neat stuff. I've been worried it'd use shuttle controller (which is getting modified in the Bay merge if memory serves) but nope.avi, so just test it if you haven't.

**Comment by Nienhaus** [comment]
I tested it last weekend and it worked.

**Comment by scrdest** [comment]
Seems fine. Glloyd should be able to review it himself today, considering he mentioned doing events so I presume he'll be on.

**Comment by Nienhaus** [comment]
wzxgpsz

**Comment by Glloyd** [comment]
Events and scom are next weekend btw. I'll have Internet by midweek

**Comment by scrdest** [comment]
Yeah, I misread the post, but I assumed everyone here realized I'm a derp.

**Comment by scrdest** [comment]
:+1:, Glloyd?

**Comment by Nienhaus** [comment]
@Glloyd That might help @scrdest

File: code/modules/tram/tram.dm [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+				del(src)
+			return
+		if(3.0)
+			if(prob(25))
+				del(src)
+			return
+
+/obj/tram/blob_act()
+	if(prob(50))
+		del(src)
```
Comment (Iamgoofball) [inline code/modules/tram/tram.dm]: >del()

Change to qdel()

Comment (Nienhaus) [inline code/modules/tram/tram.dm]: So remove the src?

Comment (Iamgoofball) [inline code/modules/tram/tram.dm]: qdel(src)

Comment (Nienhaus) [inline code/modules/tram/tram.dm]: http://i.imgur.com/8jKTeGR.png :I

Comment (Iamgoofball) [inline code/modules/tram/tram.dm]: GLLOYD UPDATE YOUR SHIT ITS 2015 ONLY CASUALS DON'T USE QDEL

Comment (Nienhaus) [inline code/modules/tram/tram.dm]: @Iamgoofball Still haven't seen it on Bay http://i.imgur.com/EDu2oZb.png :I

Comment (scrdest) [inline code/modules/tram/tram.dm]: @Iamgoofball people have this thing called 'life' and 'being busy' and if you followed the community you'd know both of those mythical, rare events occurred for Glloyd. Update is scheduled this week, IIRC, as a matter of fact.

Comment (TGameCo) [inline code/modules/tram/tram.dm]: What are the benefits of qdel over just del?

Comment (Glloyd) [inline code/modules/tram/tram.dm]: we don't have qdel yet.

On Thu, Jun 4, 2015 at 7:49 PM, Thomas Stoeckert <notifications@github.com>
wrote:

> In code/modules/tram/tram.dm
> <https://github.com/UristMcStation/UristMcStation/pull/229#discussion_r31777509>
> :
>
> > +	switch(severity)
> > +		if(1.0)
> > +			del(src)
> > +			return
> > +		if(2.0)
> > +			if(prob(50))
> > +				del(src)
> > +			return
> > +		if(3.0)
> > +			if(prob(25))
> > +				del(src)
> > +			return
> > +
> > +/obj/tram/blob_act()
> > +	if(prob(50))
> > +		del(src)
>
> What are the benefits of qdel over just del?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UristMcStation/UristMcStation/pull/229/files#r31777509>
> .
>



-- 
Graham Lloyd

**Comment by Glloyd** [comment]
also still no internet, i hate canadians isps

**Comment by Nienhaus** [comment]
♥

**Pull request merged**